apps/src/studio/apiJavascript.js

```diff
@@ -0,0 +1,105 @@
+// API definitions for functions exposed for JavaScript (droplet/ace) levels:
+
+exports.setBackground = function (value) {
+  Studio.queueCmd(null, 'setBackground', {'value': value});
+};
+
+exports.setSprite = function (spriteIndex, value) {
+  Studio.queueCmd(null, 'setSprite', {
+    'spriteIndex': spriteIndex,
+    'value': value
+  });
+};
+
+exports.setSpriteEmotion = function (spriteIndex, value) {
+  Studio.queueCmd(null, 'setSpriteEmotion', {
+    'spriteIndex': spriteIndex,
+    'value': value
+  });
+};
+
+exports.setSpriteSpeed = function (spriteIndex, value) {
+  Studio.queueCmd(null, 'setSpriteSpeed', {
+    'spriteIndex': spriteIndex,
+    'value': value
+  });
+};
+
+/*
+exports.setSpriteSize = function (spriteIndex, value) {
+  Studio.queueCmd(null, 'setSpriteSize', {
+    'spriteIndex': spriteIndex,
+    'value': value
+  });
+};
+*/
+
+exports.setSpritePosition = function (spriteIndex, value) {
+  Studio.queueCmd(null, 'setSpritePosition', {
+    'spriteIndex': spriteIndex,
+    'value': value
+  });
+};
+
+/*
+exports.setSpriteXY = function (spriteIndex, xpos, ypos) {
```

Comment:
Both are APIs that we could choose to enable (not long running blocks), but are not currently in the toolbox for the droplet level. I should make that more clear in my next change.